firebase-case-study.md

```diff
@@ -0,0 +1,133 @@
+
+
+In 2014 they launched Firebase Hosting and Firebase Authentication as two new products. Later that year, in October, Google purchased Firebase and started further development and maintenance of the Brand. Google then acquired Divshot (2015), an interface builder and web hosting platform to merge into the Firebase Team. In the following years they expanded the service further and integrated AdMob, Google Platform and Google Ads (2016), as well as acquired Fabric and Crashlytics from Twitter (2017). They’ve expanded their service through the years and there are now several options to build, improve and grow an application with Firebase. 
+
+![image](https://user-images.githubusercontent.com/91533917/218750092-e8d847d7-538b-414e-9d04-83966b1a886b.png)
+
+
+## Features
+According to Firebase’s Homepage, these are the features sorted in their respective category:
+
+Build
+- Cloud Firestore
+- Realtime Database
+- Remote Config
+- Firebase Extensions
+- App Check
+- Cloud Functions
+- Authentication
+- Cloud Messaging
+- Hosting
+- Cloud Storage
+- Firebase ML
+
+Release & Monitor
+- Crashlytics
+- Google Analytics
+- Remote Config
+- Performance Monitoring
+- Test Lab
+- App Distribution
+
+Engage 
+- Remote Config
+- Google Analytics
+- A/B Testing
+- Authentication
+- Cloud Messaging
+- Crashlytics
+- Dynamic Links
+- In-App Messaging
+
+
+## Strengths
+
+1. Faster Development, as it offers a set of pre-requisites of backend development.
+2. Has different pricing models to suit your needs.
+3. Firebase has a real-time database, which collects, processes and enriches incoming series of data points immediately after the data is created.
+4. Has incredible storage size potential for storing data.
+5. It is a serverless framework that is stored in a cloud and runs in a managed environment.
+6. Firebase is highly secure by encrypting data in transit and isolate customer data. Several of the services Firebase offer also encrypt data at rest.
+7. Has the most advanced hosted Backend-as-a-Service (BaaS) solution.
+8. It is quick and easy to set up.
+9. Firebase provides three-way data binding with AngularFire, making sure all data is updated everywhere automatically.
+10. Has simple serialization of app state.
+11. It is easy to access data, files, authorization and more.
+12. No infrastructure to server needed to power apps with data.
+13. Has JSON storage, which means no issue between data and objects.
+
+
+## Weaknesses
+
+1. There are limited querying and indexing capabilities.
+2. Has limited data migration.
+3. It does not support native aggregation.
+4. Some features are platform dependent. 
+5. Firebase is Android centered.
+6. Has less support for iOS. 
+
+
+## Comparison
```

Comment:
Great comparison to MongoDB! 

Suggestion: add a paragraph about the overall competition Firebase has, or add another comparison to another BaaS platform like Supabase.